packages/theme/src/cli/utilities/theme-fs/theme-fs-mock-factory.ts

```diff
@@ -0,0 +1,14 @@
+import {ThemeAsset, ThemeFileSystem} from '@shopify/cli-kit/node/themes/types'
+
+export function mockThemeFileSystem(root: string, files: Map<string, ThemeAsset>): ThemeFileSystem {
```

Comment:
I'd personally rename this function to `fakeThemeFileSystem` to communicate even better what it does :) 🚀